demos/link-prediction/random-walks/cora-lp-demo.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Maybe we don't need to have the same comments twice just with "graph" -&gt; "graph_test"; this cell could just have a comment like</p><p> </p><pre class="ql-syntax" spellcheck="false"># Do the same process to compute a training subset from within the test graph
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>